openssl-sys/build.rs

```diff
@@ -343,8 +343,10 @@ RUST_LIBRESSL_251
 RUST_LIBRESSL_250
 #elif defined (LIBRESSL_VERSION_NUMBER) && LIBRESSL_VERSION_NUMBER < 0x20500000
 RUST_LIBRESSL_OLD
-#elif OPENSSL_VERSION_NUMBER >= 0x10101000
+#elif OPENSSL_VERSION_NUMBER >= 0x10102000
```

Comment:
Why reject 1.1.2, given the effort they're going to to maintain ABI compat?

Reply:
Their notion of ABI compat is not all that strict. Several constants have changed values between 1.1.0 and 1.1.1. 1.1.2 isn't going to be releasing for over a year, so I don't think this check is really going to be a problem for people.

Reply:
> Several constants have changed values between 1.1.0 and 1.1.1

That's odd, they seemed pretty clear about that specific type of change being forbidden. Do you recall any specifics?

Reply:
https://github.com/sfackler/rust-openssl/pull/839/files#diff-ceca4680e3c63bdc67652c99c23c0db9

SSL_OP_NO_SSL_MASK changed in 1.1.1, and SSL_OP_ALL changed in 1.1.0f.